2020/hamming-codes/hungarian/sentence_translations.json

```diff
@@ -1369,7 +1369,7 @@
  },
  {
   "input": "What's more, the parity of the whole block is odd, giving us confidence that there was one flip and not two.",
-  "translatedText": "Ráadásul az egész blokk paritása páratlan, így biztosak lehetünk benne, hogy egy szaltó volt, és nem kettő.",
+  "translatedText": "Ráadásul az egész blokk paritása páratlan, így biztosak lehetünk benne, hogy egy bit hiba volt, és nem kettő.",
```

Comment:
```suggestion
  "translatedText": "Ráadásul az egész blokk paritása páratlan, így biztosak lehetünk benne, hogy csak két bit lehetett hibás, tehát egy hiba van.",
```

Reply:
Suggestion conveys a different meaning from the original. ("only two bits could have been faulty, so there is in fact a single error"). I'll change mine to better reflect the wording, but keep the original meaning.